dev-games/box2d/box2d-2.3.1.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="A 2D Physics Engine for Games"
+DESCRIPTION="Box2D is a feature rich 2D rigid body physics engine, written in C++. \
+It has been used in many games, including Crayon Physics Deluxe, \
+winner of the 2008 Independant Game Festival Grand Prize"
+HOMEPAGE="http://www.box2d.org"
+SOURCE_URI="https://github.com/erincatto/Box2D/archive/v2.3.1.zip"
+LICENSE="MIT"
+COPYRIGHT="2007-2015 Erin Catto"
+CHECKSUM_SHA256="67c451bbf6d9b0ceae80e90c971f963ec149c59fb6ac73d8ebbf2c58c918f423"
+REVISION="1"
+ARCHITECTURES="!x86_gcc2 ?x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+SOURCE_DIR="Box2D-2.3.1"
```

Comment:
This is all out of order. See https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering for proper formatting.